Jenkinsfile

```diff
@@ -13,6 +13,21 @@ pipeline {
                 checkout scm
             }
         }
+        stage('rust') {
+            steps {
+                sh 'curl https://sh.rustup.rs -sSf | sh -s -- -y'
+                sh 'source $HOME/.cargo/env'
+                sh '/Users/synctesting/.cargo/bin/rustup target add aarch64-apple-ios armv7-apple-ios armv7s-apple-ios x86_64-apple-ios i386-apple-ios'
+                sh '''
+                    if ! [ -x "$(command -v /Users/synctesting/.cargo/bin/cargo-lipo)" ] ; then
+                    /Users/synctesting/.cargo/bin/cargo install cargo-lipo
+                    echo "Cargo-lipo is not installed, installing" >&2
```

Comment:
perhaps we can move this echo before install, and let echo on for the cargo install command so any error msg can be shown?

Reply:
👍